packages/drivers/odsp-driver/src/localOdspDriver/localOdspDocumentStorageManager.ts

```diff
@@ -51,7 +51,7 @@ export class LocalOdspDocumentStorageService extends OdspDocumentStorageServiceB
 		let snapshotContents: ISnapshot;
 
 		if (typeof this.localSnapshot === "string") {
-			const content: IOdspSnapshot = JSON.parse(this.localSnapshot);
+			const content = JSON.parse(this.localSnapshot) as IOdspSnapshot;
```

Comment:
Bringing in some bias from my C# days and an argument against using `var` to infer the type of something based on whatever the right-hand-side evaluates to: I don't like the new pattern as much because it kind of suggests (to me at least) that if down the road this is not enough, "just change the cast to something that works" might be ok. Whereas tying the variable declaration suggests "we expect this to be an X", kind of like an invariant, and explicitly changing the type of something that was deliberately typed seems worth deeper inspection during refactoring.

Reply:
Fair. A better short term solution is probably:
```suggestion
			const content: IOdspSnapshot = JSON.parse(this.localSnapshot) as IOdspSnapshot;
```